frontend/src/metabase/dashboard/selectors/selectors-typed.ts

```diff
@@ -18,9 +18,11 @@ export function getSelectedTabId(state: State) {
   return state.dashboard.selectedTabId;
 }
 
-export const getDashcards = (state: State) => state.dashboard.dashcards;
+export function getDashcards(state: State) {
+  return state.dashboard.dashcards;
+}
 
-export const getDashCardById = (state: State, dashcardId: DashCardId) => {
+export function getDashCardById(state: State, dashcardId: DashCardId) {
```

Comment:
I touched this file when I thought I'd need selectors but decided to leave these changes for consistency anyways.